src/blueprints/tv2/value-objects/tv2-video-mixer-transition.ts

```diff
@@ -0,0 +1,29 @@
+export enum VideoMixerTransition {
+  MIX = 'MIX',
+  DIP = 'DIP',
+  WIPE = 'WIPE',
+  SPLIT_SCREEN = 'SPLIT_SCREEN',
+  STING = 'STING',
+  CUT = 'CUT',
+  DUMMY = 'DUMMY'
+}
+
+export interface VideoMixerTransitionSettings {
+  mix?: {
+    rate: number
+  }
+  dip?: {
+    rate: number,
+    input: number
+  },
+  wipe?: {
+    /** 1 - 250 frames */
+    framerate?: number;
```

Comment:
These are no longer the values used in TSR, so we can use our formatting conventions e.g. it should be `frameRate` :)

Reply:
Updated to `frameRate`, and removed comment. 
The limits are being enforced when mapping from our general use `VideoMixerTransitionSettings` to the Atem version.
Additionally a different video mixer might have different limits, thus is something that should be enforced during mappings, as i did.